karamel/include/krml/lowstar_endianness.h

```diff
@@ -77,7 +77,7 @@
 #  define le64toh(x) (x)
 
 /* ... for Windows (GCC-like, e.g. mingw or clang) */
-#elif (defined(_WIN32) || defined(_WIN64)) &&                                  \
+#elif (defined(_WIN32) || defined(_WIN64) || defined(__EMSCRIPTEN__)) &&                                  \
```

Comment:
This should be changed upstream. I filed https://github.com/FStarLang/karamel/pull/344 for that.

Reply:
thank you for filing that.